tests/functional/trace_latency_histogram.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+import pytest
+
+from api.iotrace_lat_hist_parser import LatencyHistogram
+from core.test_run import TestRun
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine, FioOutput
+from test_utils.size import Size, Unit
+from test_utils.os_utils import Udev
+
+
+@pytest.mark.parametrize("io_dir", [ReadWrite.write, ReadWrite.read])
+def test_latency_histogram_basic(io_dir):
+    """
+        title: Test for basic latency histogram properties
+        description: |
+            Test if samples count reported by fio equals count from iotracer (taking
+            into consideration dropped ones). Test if average.
+        pass_criteria:
+            - Fio's samples number equals number of iotracer samples + number of
+              dropped ones
+    """
+    iotrace = TestRun.plugins['iotrace']
+    read = ReadWrite.read is io_dir
+
+    for disk in TestRun.dut.disks:
+        with TestRun.step(f"Start tracing on {disk.system_path}"):
+            iotrace.start_tracing([disk.system_path])
+            time.sleep(3)
+        with TestRun.step(f"Run {io_dir} IO"):
+            fio = (Fio().create_command(output_type=FioOutput.jsonplus)
+                   .io_engine(IoEngine.libaio)
+                   .size(Size(1, Unit.MebiByte))
+                   .block_size(Size(1, Unit.Blocks512))
+                   .read_write(io_dir)
+                   .target(disk.system_path)
+                   .direct())
+            fio_out = fio.run()
+        with TestRun.step("Stop tracing"):
+            iotrace.stop_tracing()
+        with TestRun.step("Get latency histogram from iotracer"):
+            trace_path = iotrace.get_latest_trace_path()
+            iot_histograms = iotrace.get_latency_histograms(trace_path)
+            iot_histo = iot_histograms.read if read else iot_histograms.write
+        with TestRun.step("Build histogram from fio bins"):
+            fio_histograms = LatencyHistogram.build_histo_from_fio_job(
+                iot_histograms, fio_out[0].job)
+            fio_histo = fio_histograms.read if read else fio_histograms.write
+        with TestRun.step("Check if count of samples reported by iotracer "
+                          "(including dropped ones) equals count of fio samples "):
+            summary = iotrace.get_trace_summary(trace_path)
+            dropped_events = int(summary['droppedEvents'])
+            iot_samples_count = iot_histo.samples_count() + dropped_events
+            fio_samples_count = fio_histo.samples_count()
+            if iot_samples_count != fio_samples_count:
+                TestRun.fail(f"Wrong samples count reported by iotracer "
+                             f"({iot_samples_count}) compared to fio's "
```

Comment:
I'd break this print up into samples_count + dropped count. Might be slightly easier to check during failure